pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
By removing this you have an unknown version of pip on the host which may, or may not, understand the --constraint option. This has been the case before and broke many things, so I think this is likely to make it worse, not better.